src/main/generated/com/onlinebanking/shared/dto/mapper/UserDtoMapperImpl.java

```diff
@@ -0,0 +1,87 @@
+package com.onlinebanking.shared.dto.mapper;

```

Comment:
This file shouldn't get checked into the code base. it's a generated code.

Reply:
should we add it to the .gitignored

Reply:
No it should be as it has been since it's generated by the mapstruct and shouldn't show up but somehow it did in your git